doc/man3/SSL_load_client_CA_file.pod

```diff
@@ -66,6 +66,14 @@ The operation failed, check out the error stack for the reason.
 
 Pointer to the subject names of the successfully read certificates.
 
+=item 0
+
+The operation failed.
+
+=item 1
+
+The operation succeeded.
+
```

Comment:
I think the list should be split in two and functions returning the pointer should be listed before the first list and those returning an int should be listed before the second list.

Reply:
Unfortunately I do not think it will be acceptable with CLA: trivial though.

Reply:
> I think the list should be split in two and functions returning the pointer should be listed before the first list and those returning an int should be listed before the second list.

Done. I've addressed the review comment in the latest commit.